### ISSUE TITLE
Error when enabling react:true

### ISSUE THREAD

**Issue by GollyJer** [antfu/eslint-config#691]
### Describe the bug

Fresh install with everything @latest...
When I enable `react: true` I get the following error when eslint loads.
Other options, like `typescript: true` and `stylistic: true` don't cause an error.

```bash
[Info  - 12:42:29 AM] ESLint server is starting.
[Info  - 12:42:29 AM] ESLint server running in node v20.18.1
[Info  - 12:42:29 AM] ESLint server is running.
[Info  - 12:42:30 AM] ESLint library loaded from: C:\Users\Jeremy\label-maker\node_modules\eslint\lib\api.js
(node:14300) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
(Use `Cursor --trace-warnings ...` to show where the warning was created)
[Info  - 12:42:30 AM] Server process exited successfully
[@antfu/eslint-config] Detected running in editor, some rules are disabled.
[Error - 12:42:32 AM] An unexpected error occurred:
[Error - 12:42:32 AM] SystemError [ERR_TTY_INIT_FAILED]: TTY initialization failed: uv_tty_init returned EBADF (bad file descriptor)
    at new WriteStream (node:tty:92:11)
    at file:///C:/Users/Jeremy/label-maker/node_modules/@clack/core/dist/index.mjs:9:2141
    at new Promise (<anonymous>)
    at fD.prompt (file:///C:/Users/Jeremy/label-maker/node_modules/@clack/core/dist/index.mjs:9:1924)
    at Module.ye (file:///C:/Users/Jeremy/label-maker/node_modules/@clack/prompts/dist/index.mjs:22:7)
    at ensurePackages (file:///C:/Users/Jeremy/label-maker/node_modules/@antfu/eslint-config/dist/index.js:161:26)
    at async react (file:///C:/Users/Jeremy/label-maker/node_modules/@antfu/eslint-config/dist/index.js:1263:3)
    at async Promise.all (index 15)
    at async file:///C:/Users/Jeremy/label-maker/node_modules/eslint-flat-config-utils/dist/index.mjs:165:25
    at async FlatConfigComposer.toConfigs (file:///C:/Users/Jeremy/label-maker/node_modules/eslint-flat-config-utils/dist/index.mjs:425:17)
```

### Reproduction

export default antfu({ react: true })

### System Info

```Shell
System:
    OS: Windows 11 10.0.26100
    CPU: (16) x64 AMD Ryzen 9 PRO 7940HS w/ Radeon 780M Graphics
    Memory: 31.53 GB / 62.66 GB
  Binaries:
    Node: 22.14.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.22 - C:\Program Files\nodejs\yarn.CMD
    npm: 11.1.0 - C:\Program Files\nodejs\npm.CMD
    bun: 1.2.6 - ~\.bun\bin\bun.EXE
```

### Used Package Manager

bun

### Validations

- [x] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [x] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

### Contributions

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests (actually just go ahead and do it, thanks!)

**Comment by GollyJer** [comment]
RTFM 👍

![Image](https://github.com/user-attachments/assets/c173e104-9447-45d4-a74a-04793a481365)
